hbase-server/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupBasedLoadBalancer.java

```diff
@@ -77,6 +81,23 @@
   private volatile RSGroupInfoManager rsGroupInfoManager;
   private LoadBalancer internalBalancer;
 
+  /** Define the config key of if fallback group is enabled */
+  public static final String FALLBACK_GROUP_ENABLED_KEY = "hbase.rsgroup.fallback.groups.enable";
+  public static final boolean FALLBACK_GROUP_ENABLED_DEFAULT = false;
+
+  /** Define the config key of fallback groups */
+  public static final String FALLBACK_GROUPS_KEY = "hbase.rsgroup.fallback.groups";
```

Comment:
groups? Allow fallback to more than one group?

Reply:
Yes, we can config multiple comma-separated group.